modules/openxBidAdapter.js

```diff
@@ -16,7 +16,7 @@ export const USER_ID_CODE_TO_QUERY_ARG = {
   idl_env: 'lre', // LiveRamp IdentityLink
   lipb: 'lipbid', // LiveIntent ID
   netId: 'netid', // netID
-  parrableid: 'parrableid', // Parrable ID
+  parrableId: 'parrableId', // Parrable ID
```

Comment:
As a thought, could it be that this bidAdapter is using the key as `parrableid` and changing it to `parrableId` would cause the following value to be undefined? 

https://github.com/Parrable/Prebid.js/blob/350c0912ea6053eb45a01d52865ea888ad060ea3/modules/openxBidAdapter.js#L269

Reply:
Ah, good catch. I reviewed their code and it looks like the left hand value (key) is the one that should match our userId system name (the one that's added to the userId object). And the right value is the name they use to send off our ID in their queryParams.